client/test/e2e/shared/components.js

```diff
@@ -0,0 +1,34 @@
+/* global inject, browser, element, by, protractor */
+
+/**
+ * Hooks for the currency input component described in the component
+ * bhCurrencyInput.js.
+ * @public
+ */
+exports.currencyInput = {
+  selector : '[data-bh-currency-input]',
+
+  /**
+   * Sets the value of the currency input.
+   */
+  set : function set(value) {
+    var elm = element(by.css(this.selector));
```

Comment:
This idea of test utilities for directives is an excellent idea going forward. Very cleanly implemented.